src/FunnelChart/FunnelSeries/FunnelAxis/FunnelAxisLabel.tsx

```diff
@@ -59,38 +74,56 @@ export const FunnelAxisLabel: FC<Partial<FunnelAxisLabelProps>> = ({
   padding,
   fontSize,
   fill,
-  className
+  className,
+  position,
+  showValue,
+  alwaysShowLabel
 }) => {
   const x = xScale(index) + padding;
   const [height] = yScale.range();
   const y = height / 2 + padding;
   const label = data.key as string;
   const nextOffset = xScale(index + 1);
-  const width = (nextOffset ? nextOffset - xScale(index): 0) - padding;
+  const width = (nextOffset ? nextOffset - xScale(index) : 0) - padding;
   const size = calculateDimensions(label, fontFamily, fontSize);
 
   // If the labels don't fit, just hide them
-  if (size.width > width) {
+  if (!alwaysShowLabel && size.width > width) {
     return null;
   }
 
+  let transform: string;
```

Comment:
Let's put this in a function ( can be in scope ) to clean this up. Otherwise its easy to get this carried away.